docs/modules/ROOT/pages/integrating-with-existing-deployments.adoc

```diff
@@ -0,0 +1,240 @@
+Repeat this for every cluster you wish to add to the Cluster Monitor.
+
+=== Install the Agent on your Couchbase Server nodes
+
+See https://issues.couchbase.com/browse/CMOS-210[CMOS-210], which tracks the development (including documentation) of the Agent.
+Until this is ready, follow the instructions given to you by your Couchbase representative.
+
+== Configuration
+
+=== Grafana: Plugins
+
+[NOTE]
+====
+The instructions for each plugin offer a Grafana Cloud one-click install, a command-line based install for a running instance, or a `.zip` file which can be unpacked manually into your Grafana plugins directory. 
+====
+
+**Currently, we require only https://grafana.com/grafana/plugins/marcusolsson-json-datasource/[JSON API], which has associated https://grafana.com/grafana/plugins/marcusolsson-json-datasource/?tab=installation[installation instructions].**
+
+However, if you:
+
+- Configure Grafana (v7.1+) dashboards through the use of provisioning files: https://grafana.com/docs/grafana/latest/administration/provisioning/#plugins[specify the plugins to install in your provisioning configuration file]. 
+- Are running Grafana in a Docker container: simply pass through an additional environment variable `GF_INSTALL_PLUGINS=$PLUGIN_NAME $VERSION` where the `$PLUGIN_NAME` and latest `$VERSION` can be found on the plugin's homepage (linked under "installation instructions" for each). 
+
+=== Grafana: Dashboards
+
+The CMOS dashboards can be found in the official GitHub repository under https://github.com/couchbaselabs/observability/tree/main/microlith/grafana/provisioning/dashboards[microlith/grafana/provisioning/dashboards]/.
+Download them and copy them to a folder accessible by your Grafana installation. 
+
+The relevant setting in the Grafana configuration file, typically named `grafana.ini`, is:
+
+[source, console]
+----
+[dashboards]
+default_home_dashboard_path = <path to dashboards>/couchbase-inventory.json
+----
+
+If you are using https://grafana.com/docs/grafana/latest/administration/provisioning/[Grafana provisioning], you will need to update the `providers.options.path` argument in your provisioning configuration file (typically named `grafana.yml`) instead.
+For example:
+
+[source, console]
+----
+providers:
+    - name: dashboards
+      type: file
+      ...
+      options:
+          path: /etc/grafana/provisioning/dashboards/
+      ...
+----
+
+=== Grafana: Data Sources
+
+Now we have the required plugins and dashboards installed, we need to configure Data Sources.
+
+. **Cluster Monitor (JSON API)**, via the Web UI: click on _Add Data Source_ and select _JSON API_. 
+  - The URL should point to the Cluster Monitor with a sub-path of `/api/v1`. The port should be `:7196`.
+  - Enable `basicAuth`, with the user and password you configured.
+. **Prometheus** - this should already be configured as a Data Source.
+. **Alertmanager** - this should already be configured as a Data Source. 
```

Comment:
Not necessarily (per [Grafana's docs](https://grafana.com/docs/grafana/latest/datasources/alertmanager/), the Alertmanager data source is still in alpha, so people might not have it)

Reply:
Currently this is required as one of the prerequisites
> . **Prometheus**, with **Alertmanager** installed;

Good point that this probably shouldn't be the case, but for now is something to be CMOSed for future extension.

Reply:
Alertmanager itself is fine, but I'm referring to Alertmanager the Grafana data source - AFAICT that's still in alpha (so not enabled by default)